PopPUNK/network.py

```diff
@@ -2016,3 +2017,80 @@ def remove_non_query_components(G, rlist, qlist, use_gpu = False):
         query_subgraph = gt.GraphView(G, vfilt=query_filter)
         
     return query_subgraph, pruned_names
+
+def generate_network_from_distances(mode = None,
+                                    core_distMat = None,
+                                    acc_distMat = None,
+                                    sparse_mat = None,
+                                    previous_mst = None,
+                                    combined_seq = None,
+                                    rlist = None,
+                                    old_rlist = None,
+                                    model = None,
+                                    distance_type = 'core',
+                                    threads = 1,
+                                    gpu_graph = False):
```

Comment:
Probably don't want all as `None` default? Required arguments with no default

Reply:
Good point, fixed in https://github.com/bacpop/PopPUNK/commit/4e94a03eca89f8e36927b599e2d34959b4813c89 (and added missing argument).